README-Automation.md

```diff
@@ -0,0 +1,195 @@
+#Automation Scripts
+
+## Table of Contents
+
+- [Services](#services)
+- [Installing The Automation Scripts](#installing-the-automation-scripts)
+  - [Stable Channel](#stable-channel)
+  - [Unstable Channel](#unstable-channel)
+- [Initializing Drive](#initializing-drive)
+- [Using The Automation Scripts](#using-the-automation-scripts)
+  - [Setting Up](#setting-up)
+  - [Selecting Objects](#selecting-objects)
+  - [Launching A Script](#launching-a-script)
+  - [Help](#help)
+- [drive-menu Screenshots](#drive-menu-screenshots)
+- [Tracking And Filing Issues](#tracking-and-filing-issues)
+- [Reaching Out](#reaching-out)
+- [Disclaimer](#disclaimer)
+- [LICENSE](#license)
+
+## Services
+
+JC Manciot's [PPA](https://gitlab.com/jean-christophe-manciot/ppa) offers a collection of scripts for automated drive commands including syncing which can be used on the latest stable Ubuntu distribution.
```

Comment:
I hear you, However, Launchpad does not appear in the acronym PPA. 
Since people are familiar with what a PPA means, I use this term to differentiate between a dynamic package repository and a static one: on my [GitLab account](https://gitlab.com/dashboard/projects), there is a single dynamic repository (which one can install on Ubuntu as an apt repository) and multiple static repositories which cannot be installed as such. In the first case, the packages are signed with my PGP key. In the latter case, the packages are not signed and must be downloaded individually before being installed manually.
How about mentioning a "GitLab-hosted" PPA?